pkg/provision/zdb.go

```diff
@@ -133,6 +134,15 @@ func createZdbContainer(ctx context.Context, allocation pkg.Allocation, mode pkg
 		slog = log.With().Str("containerID", string(name)).Logger()
 	)
 
+	// sanity check
+	if len(allocation.VolumeID) < 12 {
+		return fmt.Errorf("invalid volume ID length, expecting zdb<GUID> format, got '%s'", allocation.VolumeID)
+	}
+	hw, err := hex.DecodeString(allocation.VolumeID[len(allocation.VolumeID)-12:])
```

Comment:
I don't understand how the hex of a volume can be a valid mac address ?
What should be use is this function: https://github.com/threefoldtech/zos/blob/5571c15766cf1c81a8c8c7a642fbd77d8b7b36ad/pkg/network/ifaceutil/mac.go#L11

Reply:
a mac address is just 6 bytes of data, I use the last 6 bytes of the volume GUID. so I parse the last 12 character of the string representation to 6 bytes, then just use that as MAC address

Reply:
you cannot use any 6 bytes. there are reserved range and a special bit that have to be set. That's why we have a function to do it.